### PR TITLE
Implement spinner

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/shared/components/SearchInput/SearchInput.jsx [hunk 1]
```diff
@@ -55,27 +65,36 @@ function SearchInput({ items, setChosenItem, inputName, inputPlaceholder }) {
...
+                      onClick={() => onItemsSelect(name)}
+                      text={name}
+                    />
+                  </li>
+                ))
+              )}
+            </ul>
+          ) : null}
+        </div>
+      )}
```
Comment (mi-chal) [inline src/shared/components/SearchInput/SearchInput.jsx]: Nesting multiple ternary is not a goof practice. Do something like this before main` return`
```
if ({isSearchingItems) {
  return <Spinner />
}
```

Comment (dwantula) [reply src/shared/components/SearchInput/SearchInput.jsx]: Fixed

File: src/shared/components/Spinner/styles.scss [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
+//   display: flex;
+//   width: 170px;
+//   height: 170px;
+//   flex-direction: column;
+//   justify-content: space-around;
+//   align-items: center;
+//   align-self: center;
+//   padding-left: 40px;
+//   border: 1px solid red;
+// }
```
Comment (mi-chal) [inline src/shared/components/Spinner/styles.scss]: Remove comment

File: src/shared/components/Spinner/styles.scss [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
-  height: 80px;
-  padding: 25px;
-}
-.spinner:after {
-  content: " ";
-  display: block;
-  width: 64px;
-  height: 64px;
-  margin: 8px;
+  // content: " ";
```
Comment (mi-chal) [inline src/shared/components/Spinner/styles.scss]: Remove comment

File: src/shared/components/Spinner/styles.scss [hunk 4]
```diff
@@ -1,17 +1,24 @@
...
+  display: flex;
+  align-items: center;
+  align-self: center;
+  width: 54px;
+  height: 54px;
+  margin: 15px;
   border-radius: 50%;
   border: 6px solid green;
   border-color: $green transparent $green transparent;
```
Comment (mi-chal) [inline src/shared/components/Spinner/styles.scss]: Wouldn't `border-color: $green transparent` be enough?

**Review by mi-chal — APPROVED** [review]

File: src/shared/components/Spinner/styles.scss (hunk 2 above)
Comment (dwantula) [reply src/shared/components/Spinner/styles.scss]: Done

File: src/shared/components/Spinner/styles.scss (hunk 3 above)
Comment (dwantula) [reply src/shared/components/Spinner/styles.scss]: Done

File: src/shared/components/Spinner/styles.scss (hunk 4 above)
Comment (dwantula) [reply src/shared/components/Spinner/styles.scss]: Yes, i can removed it. Done

**Pull request merged**